src/content/docs/synthetics/synthetic-monitoring/using-monitors/intro-synthetic-monitoring.mdx

```diff
@@ -27,12 +27,7 @@ Setting up synthetic monitors with New Relic lets you:
 * Get extended use case coverage with [additional Node modules](/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules) for your scripted API monitors. 
 * Easily diagnose whether an issue stems from the network or AWS location, a slow third party resource, or the health of your backend services or infrastructure. 
 
-For a short demo of how to create synthetic monitors with New Relic, watch the below video (5:10 minutes).
-
-<Video
-  id="EN5FkIW8YWk"
-  type="youtube"
-/>
+You can view community-sourced examples of some synthetic monitors in our [synthetics quickstarts](https://newrelic.github.io/quickstarts-synthetics-library/#/). These examples are open source and not supported by New Relic, but may provide valuable examples of how to maximize your synthetic monitors view examples of 
```

Comment:
I like the call out about these examples being open source and not supported by New Relic. It looks like the sentence about the examples has an extra "view examples of" at the end. Maybe we should change it to something like this:

> These examples are open source and not supported by New Relic, but contain valuable scripts and snippets that may help you cover common use cases in scripted API and scripted browser monitors.

Reply:
I like it! i'll add this in